CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # CHANGELOG
+
+## v1.13.0
+- Update APIs
```

Comment:
Let's add a link to the release notes for the corresponding changes for shopper search: https://developer.salesforce.com/docs/commerce/commerce-api/references/about-commerce-api/about.html#282024